lib/fizzy/execute.cpp

```diff
@@ -423,7 +423,7 @@ __attribute__((no_sanitize("float-divide-by-zero"))) inline constexpr T fdiv(T a
 }
 
 template <typename T>
-inline constexpr T fmin(T a, T b) noexcept
+inline T fmin(T a, T b) noexcept
```

Comment:
Both `fmin` and `fmax` use `std::signbit` which is not constexpr: https://en.cppreference.com/w/cpp/numeric/math/signbit